assets/iohub-installer

```diff
@@ -149,7 +149,7 @@ tar zxf ${firmware} -C /mnt/iohubboot2
 version2=$(cat /mnt/iohubboot2/VERSION)
 
 cd /
-echo -e "${grn}... firmware on ${disk}${devicePostfix}2 and ${disk}${devicePostfix}3 transferred${end}\n"
+echo -e "${grn}... firmware ${version1} on ${disk}${devicePostfix}2 and ${disk}${devicePostfix}3 transferred${end}\n"
```

Comment:
line 152 is not correct: there are `version1` and `version2` variables. either we use both or no one

you could add
echo -e "${grn}... firmware ${version1} onto ${disk}${devicePostfix}2 transferred${end}"
on line 146
and
echo -e "${grn}... firmware ${version2} onto ${disk}${devicePostfix}3 transferred${end}"
on line 152

and revert this modification